### PR TITLE
Add details about auto_san_validation behavior

### PR DESCRIPTION
**Pull request by kfaseela**
## Description

Add a warning about auto_sni and auto_san_validation behavior which is now enabled by default

## Reviewers

- [ ] Ambient
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

### CODE REVIEW

File: content/en/boilerplates/auto-san-validation.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+---
+Istio has `auto_sni` and `auto_san_validation` enabled by default. This means, whenever there is no explicit `sni` set in your `DestinationRule`, transport socket SNI for new upstream connections will be set based on the downstream HTTP host/authority header. And if there are no `subjectAltNames` set in the `DestinationRule` when `sni` is unset, `auto_san_validation` will kick in, and the upstream presented certificate for new upstream connections will be automatically validated based on the downstream HTTP host/authority header.
```
Comment (craigbox) [inline content/en/boilerplates/auto-san-validation.md:3]:
```suggestion
Istio has `auto_sni` and `auto_san_validation` enabled by default. This means, whenever there is no explicit `sni` set in your `DestinationRule`, transport socket SNI for new upstream connections will be set based on the downstream HTTP host/authority header. If there are no `subjectAltNames` set in the `DestinationRule` when `sni` is unset, `auto_san_validation` will kick in, and the upstream-presented certificate for new upstream connections will be automatically validated based on the downstream HTTP host/authority header.
```

Do you want to move the {{< warning >}} into the boilerplate?

Comment (kfaseela) [reply content/en/boilerplates/auto-san-validation.md]: > Do you want to move the {{< warning >}} into the boilerplate?

I changed the warning to tip based on an offline review comment from @leosarra , have moved to the boilerplate but let us see if lint complains

**Review by leosarra — APPROVED** [review]
LGTM

**Review by craigbox — APPROVED** [review]

**Pull request merged**